src/view/drag-handle/util/create-visibility-event.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import type { EventBinding } from './event-types';
+
+const getVisibiltyEvent = () : ?{hidden:string, name:string} => {
+  const prefixes = ['ms', 'webkit', 'moz', 'o'];
+  let browserPrefix;
```

Comment:
I think we can just use a `find` here (we are already using .find around the place). Also, I think we should check for the default first:

```js
// using memoizeOne so that we do not recompute this function after initial computation
const getVisibilityEvent = memoizeOne() : ?Pair => {
 // non prefixed event is supported
 if('hidden' in document) {
   return {
    hidden: 'hidden',
    name: 'visibilitychange',
  }

  const prefix: ?string = ['ms', 'webkit', 'moz', 'o'].find((prefix: string): boolean => `${prefix}Hidden` in document);

   if(!prefix) {
      return null;
   }

   return {
     hidden: `${prefix}Hidden`,
     name: `${prefix}visibilitychange`,
   }
 }
})

```

Reply:
```js
const defaultPair: Pair = {
    hidden: 'hidden',
    name: 'visibilitychange',
  }

// using memoizeOne so that we do not recompute this function after initial computation
const getVisibilityEvent = memoizeOne() : ?Pair => {
 // non prefixed event is supported
 if('hidden' in document) {
   return defaultPair;

  const prefix: ?string = ['ms', 'webkit', 'moz', 'o'].find((prefix: string): boolean => `${prefix}Hidden` in document);

   // if no prefixed event is supported - simply return the defaultPair. It will not cause any errors to bind to this event - it just won't do anything on visibility change
   if(!prefix) {
      return defaultPair;
   }

   return {
     hidden: `${prefix}Hidden`,
     name: `${prefix}visibilitychange`,
   }
 }
})
```